### PR TITLE
docs: notify about 0.5-compat in the readme

### PR DESCRIPTION
**Pull request by vigoux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Please consider the experience with this plug-in as experimental until Neovim 0.
 You can find the current roadmap [here](https://github.com/nvim-treesitter/nvim-treesitter/projects/1).
 The roadmap and all features of this plugin are open to change, and any suggestion will be highly appreciated!**
 
+*If you want to use this plugin with Neovim 0.5, please use the
+0.5-compat branch*
+
```
Comment (stsewd) [inline README.md:46]: Maybe add a note that some improvements and bug fixes may require using neovim nightly?

Comment (vigoux) [reply README.md:47]: Done

**Review by theHamsta — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -41,6 +41,10 @@ Please consider the experience with this plug-in as experimental until Neovim 0.
 You can find the current roadmap [here](https://github.com/nvim-treesitter/nvim-treesitter/projects/1).
 The roadmap and all features of this plugin are open to change, and any suggestion will be highly appreciated!**
 
+*If you want to use this plugin with Neovim 0.5, please use the
+0.5-compat branch. Be aware though that most improvements will require
+neovim nightly.*
+
```
Comment (theHamsta) [inline README.md:47]: Maybe also put the 0.5 install command next to the current one?

**Review by stsewd — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (vigoux) [reply README.md:47]: I'd say we split them, just to make them separate and attract peoples eyes

**Pull request merged**